sympy/physics/units/tests/test_quantities.py

```diff
@@ -510,3 +512,32 @@ def test_issue_22164():
 
     # deprecation warning is not expected here
     SI._collect_factor_and_dimension(expr)
+
+def test_prefixed_flag():
```

Comment:
Maybe `test_prefixed_property`, I wouldn't call it a `flag` as it's supposed to be a constant.